internal/ingress/annotations/authtls/main.go

```diff
@@ -48,6 +50,7 @@ type Config struct {
 	ErrorPage          string `json:"errorPage"`
 	PassCertToUpstream bool   `json:"passCertToUpstream"`
 	MatchCN            string `json:"matchCN"`
+	SendClientCa       string `json:"SendClientCa"`
```

Comment:
```suggestion
	SendClientCA       string `json:"sendClientCA"`
```
I suggest modifying their style